keycloak4s-admin/src/main/scala/com/fullfacing/keycloak4s.admin/services/Roles.scala

```diff
@@ -17,9 +18,9 @@ class Roles[R[+_]: Concurrent, S](implicit client: KeycloakClient[R, S]) {
 
     // --- CRUD --- //
     /** Create a client level role for the given client. */
-    def create(clientId: UUID, role: Role.Create): R[Either[KeycloakError, Unit]] = {
+    def create(clientId: UUID, role: Role.Create): R[Either[KeycloakError, String]] = {
       val path: Path = Seq(client.realm, "clients", clientId, "roles")
-      client.post[Unit](path, role.copy(clientRole = true))
+      Concurrent[R].map(client.post[Headers](path, role.copy(clientRole = true)))(extractStringFromResponse)
```

Comment:
When you create `Role`, the id also should be a `UUID` right?

Reply:
Role's Id field is not restricted to UUID's. I initially had it as such but our integration tests proved me wrong.